demo/src/app/components/+collapse/demos/toggle-manual/toggle-manual.component.html

```diff
@@ -0,0 +1,11 @@
+<button type="button" class="btn btn-primary"
+        (click)="hideContent()">Show content
+</button>
+<button type="button" class="btn btn-primary"
+        (click)="showContent()">Hide content
+</button>
+<hr>
+<div *ngIf="showSelected"
```

Comment:
Why do we need a basic demo with `*ngIf` that doesn't even use CollapseDirective?